lib/pubsub/subscription.js

```diff
@@ -311,9 +314,10 @@ Subscription.prototype.delete = function(callback) {
  * subscription.pull(function(err, messages) {
  *   // messages = [
  *   //   {
- *   //     ackId: '', // ID used to acknowledge its receival.
- *   //     id: '',    // Unique message ID.
- *   //     data: ''   // Contents of the message.
+ *   //     ackId: '',     // ID used to acknowledge its receival.
```

Comment:
Each message has a single `ackId` but when you make the  `acknowledge` API call, you can acknowledge many `ackIds`. Here is the format: https://cloud.google.com/pubsub/reference/rest/v1beta2/projects/subscriptions/pull#google.pubsub.v1beta2.ReceivedMessage